app/views/users/show.html.erb

```diff
@@ -1,4 +1,57 @@
-<div class="container">
-  <h2><%= @user.email %>'s Profile</h2>
+<div class="">
 
+  <div class="row">
+  <div class="col-sm-4">
+    <div class="box center dashprofile">
+      <img class="profpiccircle" src="http://api.randomuser.me/portraits/med/women/39.jpg">
+      <br><br>
+      <h3>Gwen Fairfax</h3>
+      <h4>Harvard College</h4>
+      <p><span class="glyphicon glyphicon-user" aria-hidden="true"></span>  Student</p>
+      <br><br>
+      <div class="totalpoints col-centered">
+        <h2 class="vcenter" style="color:white">76</h2>
```

Comment:
Let's try to avoid inline styles; instead, let's use a class (make a new one if need be).